README.md

```diff
@@ -135,7 +135,7 @@ I would like to thank to my coding partners.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
```

Comment:
![image](https://github.com/ibugithub/math-Magician/assets/93039359/44be2940-19f6-4fbb-829b-55d2d1fb2437)

**[OPTIONAL]**

- [ ] It would be nice if you remove the unnecessary data from your readme file to make your readme descriptive and professional 👍